README.md

```diff
@@ -30,15 +30,15 @@ Cloning a project is similar to downloading a project. By cloning this project f
 
 1. Navigate to YOUR Repository on Github (You're already there).
 2. Click on the green code button. Copy this URL to your clipboard using Command and C or hit the handy button to the right of the URL that does it for you.
-3. Open up terminal
+3. Open up terminal.
 4. Navigate to you're intended directory. To navigate to the desktop type cd Desktop/
 5. Type: ```git clone [URL GOES HERE]```
-6. Now that you have cloned the project down to your machine we need to navigate to inside that project. To do this type: ```cd Github-Practice/```
+6. Now that you have cloned the project down to your machine we need to navigate inside that project. To do this type: ```cd Github-Practice/```
 7. To confirm you have done these steps correctly when typing: ```git status``` it should tell you that you are on a specific branch.
 
 ### Create a new branch
 
-It's best practice to work off a develop branch, not master. 
+It's best practice to work off a develop branch, not master.
```

Comment:
Could you change this from master to main?

Reply:
Changed this! This PR should reflect those changes.